cves/CVE-2014-7948.yml

```
@@ -64,16 +69,19 @@ fixes:
 - :commit: 3ec982d993b769efc893d4f3a0fa28eea95b69d0
   :note: ''
 vccs:
-- :commit:
-  :note: 
+- :commit: ''
+  :note: |
+    There is no VCC for this vulnerability. The issue is more of an exploit of current logic instead of a commit that
+    introduces a way for attackers to exploit a bug. Additionally there was no revision done otherwise a revision link would
+    have also been provided. The fix was adding code for the most part.
 upvotes_instructions: |
   For the first round, ignore this upvotes number.
 
   For the second round of reviewing, you will be giving a certain amount of
   upvotes to each vulnerability you see. Your peers will tell you how
   interesting they think this vulnerability is, and you'll add that to the
   upvotes score on your branch.
-upvotes: 
+upvotes:
```

+2 upvotes from me (see my suggestion)
```suggestion
upvotes: 2
```